tests/conftest.py

```diff
@@ -484,3 +489,41 @@ def __call__(self, dag_id='test_dag', session=None, **kwargs):
             return self
 
     return DagFactory()
+
+
+@pytest.fixture
+def get_dummy_dag(dag_maker):
```

Comment:
```suggestion
def create_dummy_dag(dag_maker):
```

Probably? It can stay as it is too -- no strong opinion

Reply:
Can you add some description/docstring on all fixtures too so a new author/contributor knows when to use what etc

Reply:
Cool.

Reply:
Description added